st2api/st2api/config.py

```diff
@@ -28,10 +28,23 @@
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
+def parse_args(args=None):
+    CONF(args=args, version=VERSION_STRING)
+
+
+def register_opts():
+    _register_common_opts()
+    _register_app_opts()
+
+
 def _register_common_opts():
     common_config.register_opts()
 
 
+def get_logging_config_path(service='api'):
```

Comment:
The argument is not used. Why do we need it passed in? Same comment applies to other services.